### PR TITLE
Port to mapzen.js

### PR DESCRIPTION
**Pull request by louh**
This removes the deprecated [mapzen-ui](https://github.com/mapzen/ui) bundle in favor of Mapzen.js (standalone) to provide the basic map UI functionality. It should be about 1/2 the size of the previous bundle, and imports Leaflet and Tangram separately, as before.

Another improvement here is that the `?debug=true` query parameter now loads unminified libraries of the entire stack, not just Tangram.

Two other small tweaks: the GitHub button up top has been hidden because it didn't make sense in this context, and the attribution in the lower right has been cleaned up (to avoid repeating the "Mapzen", a problem we couldn't overcome in the old bundle).

~~One last thing that's not taken care of (which is why "do not merge") is that mapzen.js introduces its own lat/lng/zoom query params scheme. Previously, using `leaflet-hash`, the scheme is a url fragment `#zoom/lat/lng`, with mapzen.js this is now part of the query string parameters. Backwards compatibility between these has not yet fully materialized, but it should be done before merge.~~

It is good enough to examine otherwise, @meetar would you mind kicking the tires and see if I missed anything? It is testable on Precog https://precog.mapzen.com/tangrams/tangram-frame/lou/mapzenjs/#lat=40.7053&lng=-74.0098&z=15

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,6 +90,21 @@
...
+        }
+
+        @media only screen and (max-width: 400px) {
+            .mz-bug-container {
+                width: 144px;
+            }
+        }
+
+        .mz-bug-github-link {
+            display: none;
```
Comment (meetar) [inline index.html:105]: What was the context where the github link didn't make sense? (In general I've really liked the ability to link straight back to the source…)

File: main.js [hunk 2]
```diff
@@ -225,85 +228,106 @@ function initLeaflet() {
...
+            // #[zoom],[lat],[lng]
+            var url_hash = window.location.hash.slice(1, window.location.hash.length).split('/');
+
+            if (url_hash.length == 3) {
+                map_start_location = {
+                    lat: Number(url_hash[1]),
+                    lng: Number(url_hash[2]),
+                    zoom: Number(url_hash[0])
+                };
+            }
```
Comment (meetar) [inline main.js]: So originally we made the call for Tangram to use (as somebody requested long ago) a "bog-standard leaflet hash" which would copy and paste easily between Tangram maps and other Leaflet maps… is the query-based style preferable? (Or are we planning to switch everywhere?)

**Review by meetar — COMMENTED** [review]
Looks great! I have just a couple of questions on topics about which I hold only very mild opinions

File: index.html (hunk 1 above)
Comment (louh) [reply index.html:105]: The GitHub link on this was always "https://github.com/mapzen/" - a default value, not really the source of what you were looking at.

**Comment by meetar** [comment]
Gotcha, thanks for the clarification.

**Comment by louh** [comment]
For future reference: I had automatically adopted the mapzen.js hash scheme, but this is optional, so we don't actually need to use it. As a result, we are not migrating off the common leaflet-hash library here.

File: index.html (hunk 1 above)
Comment (meetar) [inline index.html]: Gotcha

**Review by meetar — APPROVED** [review]
Seems reasonable :+1:

**Pull request merged**